File: test3.py
def is_leap_year(year):
    num = int(year)

    if((num % 4 == 0 and num % 100 != 0) or num % 400 == 0):
        print('Year {} is a leap year'.format(year))
    else:
        print('Year {} is not a leap year'.format(year))


is_leap_year(2020)
is_leap_year(2022)

File: test3.js
function isLeapYear(year) {
    let numYear = Number(year);
    if ((numYear % 4 === 0 && numYear % 100 !== 0) || numYear % 400 === 0) {
        console.log(`Year ${year} is a leap year`);
        return `Year ${year} is a leap year`;
    } else {
        console.log(`Year ${year} is not a leap year`);
        return `Year ${year} is not a leap year`;
    }
}

isLeapYear(2020);
isLeapYear(2000);
